### PR TITLE
Some improvements

### PR DESCRIPTION
**Pull request by BrightDV**
Hello

I have added some improvements:
- Now, when clicking the title/album in the player screen, there is a 'View artist' section. 

Screenshot_20211210-083745

I used the same `AlbumSearchPage` class as in the search page's artists results. It searches for the artist name and shows a list of all found.

Screenshot_20211210-083811

- When clicking 'Search on YouTube' in the player screen, it searches the music in the YouTube part of the app to keep the user in the app instead of using url_launcher to open it in a web browser.

Screenshot_20211210-083819

I used the `YouTubeSearchPage` class to do so.

Screenshot_20211210-083826

Also, there was a lot of useless import which kept the Flutter's analyze crashing, maybe due to the new version of Flutter?
Normally, I have fixed them!

### CODE REVIEW

**Comment by Sangwan5688** [comment]
Ok so I haven't checked the code yet but the idea of view artist is fine. But as for youtube one. The thing is it's not "Search on youtube", it's "search video" or "watch video (when accesssed from yt section)". It's there so that if someone want to watch video of that particular song so he/she can watch the video there. It was never supposed to search song inside app itself.

**Comment by BrightDV** [comment]
>It was never supposed to search song inside app itself.

Ah, ok, I see. Sorry for that.

File: lib/Screens/Player/audioplayer.dart [hunk 1]
```diff
@@ -375,10 +375,15 @@ class _PlayScreenState extends State<PlayScreen> {
                           );
                         }
                         if (value == 3) {
-                          launch(
-                            mediaItem.genre == 'YouTube'
-                                ? 'https://youtube.com/watch?v=${mediaItem.id}'
-                                : 'https://www.youtube.com/results?search_query=${mediaItem.title} by ${mediaItem.artist}',
+                          Navigator.push(
```
Comment (Sangwan5688) [inline lib/Screens/Player/audioplayer.dart:378]: Just revert this back. Rest is fine

**Comment by BrightDV** [comment]
It should be good now

**Comment by Sangwan5688** [comment]
Cool, looks fine

**Pull request merged**